Dockerfile

```
@@ -1,15 +1,19 @@
-FROM node:16.20.0
+# Bullseyes uses Debian 11 with longer end-of-life date. Slim variant means smaller software component
+FROM node:16.20.0-bullseye-slim as build
 
-WORKDIR /usr/src/app
+RUN apt-get update && apt-get install -y --no-install-recommends dumb-init
+
+ENV NODE_ENV production
 
-COPY package*.json ./
+WORKDIR /usr/src/app
 
-RUN npm install
+COPY --chown=node:node . /usr/src/app
```

Couldn't this be
```suggestion
COPY --chown=node:node . .
```
?